tldr.l

```diff
@@ -114,7 +114,7 @@ space [ \t]
   if (this.topState() === 'example_description') {
     this.popState();
     yytext = this.matches[1];
-    if (!yytext.match(/^[[A-Z]/)) yy.error(yylloc, 'TLDR003');
+    if (!yytext.match(/^[A-Z[]/)) yy.error(yylloc, 'TLDR015');
```

Comment:
Should the additional bracket be escaped here, or is that not necessary?
```suggestion
    if (!yytext.match(/^[A-Z\[]/)) yy.error(yylloc, 'TLDR015');
```

Reply:
is not necessary in this case because you can't nested `[]`

Reply:
Cheers, just wanted to make sure. 👍🏻